homu/main.py

```diff
@@ -75,6 +75,7 @@ def head_advanced(self, head_sha, *, use_db=True):
         self.head_sha = head_sha
         self.approved_by = ''
         self.status = ''
+        self.delegate = ''
```

Comment:
`head_advanced` is called whenever the HEAD of a PR is changed by the author, which means more commits are pushed to the PR. So, `self.delegate` would also be cleared whenever a PR is changed. However, I think it would be better to keep it after each PR change. Therefore the appropriate location of the initialization might be around line 59, e.g. `delegate = ''`.